control-plane/agents/common/src/msg_translation/mod.rs

```diff
@@ -0,0 +1,5 @@
+/// translate between message bus and v0 gRPC type
```

Comment:
there's no more message bus right?

Reply:
Ahh right, I will change it to `transport` type.